os/kernel/task_monitor/Kconfig

```diff
@@ -22,9 +22,16 @@ config TASK_MONITOR_PRIORITY
 
 config TASK_MONITOR_INTERVAL
 	int "Interval for checking alive(sec)"
-	default 10
+	default 5
 	---help---
 		Task Monitor checks the registered tasks/pthreads active status
```

Comment:
I think we'd better explain about the interval. Nobody can know the difference between TASK_MONITOR_INTERVAL and TASK_MONITOR_MAX_INTERVAL.